src/base/bittorrent/session.cpp

```diff
@@ -378,6 +378,7 @@ Session::Session(QObject *parent)
     sessionSettings.connection_speed = 20; // default is 10
     sessionSettings.no_connect_privileged_ports = false;
     sessionSettings.seed_choking_algorithm = libt::session_settings::fastest_upload;
+    sessionSettings.use_disk_cache_pool = false;
```

Comment:
Please add comment about this line with some description and with link to affected GitHub issue.